libs/application/templates/inheritance-report/src/forms/sections/assets.ts

```diff
@@ -435,7 +452,7 @@ export const assets = buildSection({
                   },
                   {
                     title: m.stocksNationalId,
-                    id: 'assetNumber',
+                    id: 'nationalId',
```

Comment:
the mapper is very naive and just listens to the original properties listed by DC. This may break the intended pre-fill functionality. Maybe go with `assetNumber`, unless this is fine :)